niet/__main__.py

```diff
@@ -1,15 +1,76 @@
 #!/usr/bin/python
 import argparse
 import json
-import sys
 import os.path
+import sys
+from future.types import str
```

Comment:
python2.7 and python3.x can deal with `str` don't need to use `futur.types` for this.
```shell
$ python2.7 -c 'test="ok";print(isinstance(test, str))'
True
$ python3.5 -c 'test="ok";print(isinstance(test, str))'
True
```